05_ticket_v2/05_nullability code example does not compile/run

Chapter 5.5 Nullability

https://rust-exercises.com/100-exercises/05_ticket_v2/05_nullability#tuples

Clicking the run icon produces:

```
   Compiling playground v0.0.1 (/playground)
error[E0425]: cannot find value `second` in this scope
 --> src/main.rs:3:12
  |
3 | assert_eq!(second.0, -42);
  |            ^^^^^^ not found in this scope

error[E0425]: cannot find value `second` in this scope
 --> src/main.rs:4:12
  |
4 | assert_eq!(second.1, 3);
  |            ^^^^^^ not found in this scope

error[E0425]: cannot find value `second` in this scope
 --> src/main.rs:5:12
  |
5 | assert_eq!(second.2, 8);
  |            ^^^^^^ not found in this scope

For more information about this error, try `rustc --explain E0425`.
error: could not compile `playground` (bin "playground") due to 3 previous errors
```

The code right above is not included in the snippet. It should be included, but hidden.

--- a/book/book.toml
+++ b/book/book.toml
@@ -18,6 +18,9 @@ after = ["exercise-linker"]
 [output.html]
 git-repository-url = "https://github.com/mainmatter/100-exercises-to-learn-rust"
 
+[output.html.playground]
+runnable = false
+
 [output.pandoc]
 optional = true
 hosted-html = "https://rust-exercises.com/100-exercises/"

